client/src/ui/molecules/document-survey/surveys.ts

```diff
@@ -93,14 +95,18 @@ export const SURVEYS: Survey[] = [
     end: Infinity,
   },
   {
-    key: SurveyKey.HOMEPAGE_FEEDBACK_2024,
-    bucket: SurveyBucket.HOMEPAGE_FEEDBACK_2024,
-    show: (doc: Pick<Doc, "mdn_url">) => /^\/[^/]+\/$/i.test(doc.mdn_url),
-    src: "https://survey.alchemer.com/s3/8075407/MDN-Homepage-Improvements",
-    teaser: "We are refreshing our homepage and would love",
-    question: "your input",
-    ...survey_duration(SurveyBucket.HOMEPAGE_FEEDBACK_2024),
-    ...survey_rates(SurveyKey.HOMEPAGE_FEEDBACK_2024),
+    key: SurveyKey.HOUSE_SURVEY_2025,
+    bucket: SurveyBucket.HOUSE_SURVEY_2025,
+    show: (doc: Pick<Doc, "mdn_url">) =>
+      /^\/[^/]+\/docs\/(Web|Learn_web_development)($|\/.*$)|/i.test(
+        doc.mdn_url
+      ),
+    src: "https://survey.alchemer.com/s3/8145183/MDN-short-survey",
+    teaser:
+      "We'd love to hear more about your role and the company you work for",
```

Comment:
Aren't we missing punctuation here? (cc @Rumyra)

```suggestion
      "We'd love to hear more about your role and the company you work for!",
```

Reply:
The other recent surveys didn't have punctuation either, so going with this version for now.